### PR TITLE
Update documentation from py2 to py3

### LINKED ISSUE #1772: Installation instructions to build from source need updating
### Bug Description
The documentation to build from source still says to download Python 2.7, not Python 3.7+.
http://reactionmechanismgenerator.github.io/RMG-Py/users/rmg/installation/anacondaDeveloper.html#anacondadeveloper
this causes confusing problems for a new user.

### PR DESCRIPTION
**Pull request by hwpang**
### Motivation or Problem
Update documentation from py2 to py3 and fix some wrong import directory.

### Description of Changes
A clear and concise description of what what you've changed or added.

### Testing
After changes, `make html` builds `build/html/` for RMG-Py3 successfully.

### Reviewer Tips
Suggestions for verifying that this PR works or other notes for the reviewer.

fixes #1772

### CODE REVIEW

File: rmgpy/molecule/atomtype.py [hunk 1]
```diff
@@ -69,22 +69,22 @@ class AtomType:
...
+    `quadruple`           ``list``            The total number of quadruple bonds on the atom
+    `benzene`             ``list``            The total number of benzene bonds on the atom
+    `lone_pairs`          ``list``            The number of lone pairs on the atom
+    `charge`              ``list``            The partial charge of the atom
     ===================== =================== ====================================
 
     """
 
-    def __init__(self, label='',
+    def __init__(self, label=``,
```
Comment (mliu49) [inline rmgpy/molecule/atomtype.py]: I think you probably didn't intend to replace all `'` with `` ` `` in this file?

Comment (hwpang) [reply rmgpy/molecule/atomtype.py]: @mliu49 I only intended to change the `'` in the docstring block. I've recovered those that should have remained `'`.

**Comment by amarkpayne** [comment]
Thanks for the PR @hwpang ! I hope you don't mind I have pushed two additional commits that update the installation section of the documentation now that we are using python 3.

**Comment by ajocher** [comment]
Thank you! I hope this is the right pull request to update the description for the RMG-database scripts with Python 3?

**Comment by amarkpayne** [comment]
> Thank you! I hope this is the right pull request to update the description for the RMG-database scripts with Python 3?

Yes, thanks @ajocher !

File: documentation/source/users/rmg/installation/anacondaDeveloper.rst [hunk 2]
```diff
@@ -29,7 +29,7 @@ Installation by Source Using Anaconda Environment for Unix-based Systems: Linux
     git clone https://github.com/ReactionMechanismGenerator/RMG-Py.git
     git clone https://github.com/ReactionMechanismGenerator/RMG-database.git
 
-#. Now create the anaconda environment for RMG-Py
+#. Now create the conda environment for RMG-Py
```
Comment (mliu49) [inline documentation/source/users/rmg/installation/anacondaDeveloper.rst]: The names of the environment files should also be updated below.

File: documentation/source/users/rmg/installation/index.rst [hunk 3]
```diff
@@ -5,7 +5,7 @@ Installation
 ************
 
 .. NOTE::
-    It is recommended that RMG be installed with Python 2.7, although it has been previously tested that Python 2.5 and 2.6 may also work. Dependency issues render it incompatible with Python 3.x releases.
+    It is recommended that RMG be installed with Python 3.7. Legacy releases compatible with Python 2.7 are also available
```
Comment (mliu49) [inline documentation/source/users/rmg/installation/index.rst]: Period at the end

File: documentation/source/users/rmg/installation/anacondaDeveloper.rst (hunk 2 above)
Comment (amarkpayne) [reply documentation/source/users/rmg/installation/anacondaDeveloper.rst]: Ah, sorry I forgot to make this change as well. I am pushing a fix-up commit with this change. Thanks for the catch!

**Comment by mliu49** [comment]
I pushed a few more fixes to the documentation. Please review the new changes and squash when ready. I think the other changes look good.

**Comment by mliu49** [comment]
I squashed and rebased on master. The last two commits are new.

**Pull request merged**